src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -115,7 +117,10 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
         if args.wma_disablewriterecovery:
             copyCommand += "env XRD_WRITERECOVERY=0 "
 
-        copyCommand += "%s --force --nopbar " % xrdcpExec
+        if args.wma_preload:
+            self.preload = "%s " % args.wma_preload
+
+        copyCommand += "%s%s --force --nopbar " % (self.preload, xrdcpExec)
```

Comment:
I think all these commands would look cleaner if we add an extra space in every single command, instead of stucking two format strings altogether. So, something like:
```
copyCommand += "%s %s --force --nopbar " % (self.preload, xrdcpExec)
```
and the same to all the other changes below.

That also means there is no need for the extra space above in L121.

Reply:
If you add the extra space you break the unit tests since the "no preloader set" option now looks different from before (it has the empty space).

Reply:
Only other option I see is to have a conditional for every line that includes the preloader and make one version with preloader and one version without.

Reply:
Or we merge ignoring the unit test failures since after the merge the unit tests should work again (the version with the empty space becomes the new default).

Reply:
Perhaps we could return the stage out command like:
```
return copyCommand.strip()
```
? That should resolve the unit test failures.